iron-flex-layout.html

```diff
@@ -58,6 +58,17 @@
 @pseudoElement iron-flex-layout
 @demo demo/index.html
 -->
+<script>
+  // TODO (emarquez): remove when we have a fleshed out /deep/ solution
+  // required for backwards compat for projects that rely on a global [hidden[
+  (function() {
+    if (Polymer.Settings && !Polymer.Settings.useNativeShadow) {
```

Comment:
`useNativeShadow` is available only in polymer 1.x - should this be `useShadow` instead?